src/lib/Containers/__tests__/Artist-tests.tsx

```diff
@@ -6,46 +6,39 @@ import { Artist } from "../Artist"
 
 describe("availableTabs", () => {
   it("returns nothing if artist has no metadata, shows, or works", () => {
-    const artist = new Artist()
```

Comment:
new TS forces the props check into the class constructor - this might be fixing the "constructor must include the Props interface"  bug we've been seeing